crates/broker/src/market_monitor.rs

```diff
@@ -56,8 +57,9 @@ where
         provider: Arc<P>,
         db: DbObj,
         chain_monitor: Arc<ChainMonitorService<P>>,
+        submitter_addr: Address,
```

Comment:
`submitter_addr` makes me think this is the Proof Requestor. `prover_addr` feels more accurate

Reply:
changed